flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/history/HistoryServerArchiveFetcher.java

```diff
@@ -211,7 +215,7 @@ public void run() {
 						}
 					}
 					if (updateOverview) {
-						updateJobOverview(webDir);
+						updateJobOverview(webOverviewDir, webDir);
 					}
```

Comment:
Could not figure out the function of `numFinishedPolls.countDown();`in line 224, seems that `numFinishedPolls`   is useless as it was initialized 0

Reply:
In test case, it's quite clear this is make the HS has polled file before read, That is to say adding `numFinishedPolls` is just for test purpose, am i right ？

Reply:
yes.